parachains/runtimes/assets/westmint/src/weights/pallet_xcm.rs

```diff
@@ -0,0 +1,116 @@
+// This file is part of Cumulus.
+
+// Cumulus is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Cumulus is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Autogenerated weights for `pallet_xcm`
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2022-12-08, STEPS: `50`, REPEAT: 20, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! HOSTNAME: `bm3`, CPU: `Intel(R) Core(TM) i7-7700K CPU @ 4.20GHz`
+//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("westmint-dev"), DB CACHE: 1024
+
+// Executed Command:
+// /home/benchbot/cargo_target_dir/production/polkadot-parachain
+// benchmark
+// pallet
+// --steps=50
+// --repeat=20
+// --extrinsic=*
+// --execution=wasm
+// --wasm-execution=compiled
+// --heap-pages=4096
+// --json-file=/var/lib/gitlab-runner/builds/zyw4fam_/0/parity/mirrors/cumulus/.git/.artifacts/bench.json
+// --pallet=pallet_xcm
+// --chain=westmint-dev
+// --header=./file_header.txt
+// --output=./parachains/runtimes/assets/westmint/src/weights/
+
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::{traits::Get, weights::Weight};
+use sp_std::marker::PhantomData;
+
+/// Weight functions for `pallet_xcm`.
+pub struct WeightInfo<T>(PhantomData<T>);
+impl<T: frame_system::Config> pallet_xcm::WeightInfo for WeightInfo<T> {
+	// Storage: PolkadotXcm SupportedVersion (r:1 w:0)
+	// Storage: PolkadotXcm VersionDiscoveryQueue (r:1 w:1)
+	// Storage: PolkadotXcm SafeXcmVersion (r:1 w:0)
+	// Storage: ParachainSystem HostConfiguration (r:1 w:0)
+	// Storage: ParachainSystem PendingUpwardMessages (r:1 w:1)
+	fn send() -> Weight {
+		// Minimum execution time: 26_652 nanoseconds.
+		Weight::from_ref_time(27_278_000)
+			.saturating_add(T::DbWeight::get().reads(5))
+			.saturating_add(T::DbWeight::get().writes(2))
+	}
+	// Storage: ParachainInfo ParachainId (r:1 w:0)
+	fn teleport_assets() -> Weight {
+		// Minimum execution time: 32_711 nanoseconds.
+		Weight::from_ref_time(33_447_000)
+			.saturating_add(T::DbWeight::get().reads(1))
+	}
+	// Storage: ParachainInfo ParachainId (r:1 w:0)
+	fn reserve_transfer_assets() -> Weight {
+		// Minimum execution time: 24_850 nanoseconds.
+		Weight::from_ref_time(25_427_000)
+			.saturating_add(T::DbWeight::get().reads(1))
+	}
+	fn execute() -> Weight {
+		// Minimum execution time: 14_909 nanoseconds.
+		Weight::from_ref_time(15_229_000)
```

Comment:
You'll notice that westmint is the only runtime that doesn't have `Weight::MAX` here, that means that all other runtimes have disabled any origin from running `XcmExecuteFilter`, and westmint is the only one that did enable `execute`. Is this true?

Reply:
yes, it is intentional, I checked with CGP team